config/master.key

```diff
@@ -0,0 +1 @@
+f8e0803e94fe9b7a796ab6a7ce6e2c4e
```

Comment:
dont push up the master key

put it in the gitignore and find a way to generate a new one

Reply:
**Followed these instruction to regenerate it:**

No key regeneration feature at the moment. We have to do it manually.

1)Copy content of original credentials rails credentials:show somewhere temporarily.
2)Remove config/master.key and config/credentials.yml.enc
3)Run EDITOR=vim rails credentials:edit in the terminal: This command will create a new 4)master.key and credentials.yml.enc if they do not exist.
5)Paste the original credentials you copied (step 1) in the new credentials file (and save + 
   quit vim)
6)Add and Commit the file config/credentials.yml.enc